frontend/src/components/CreateNewPEN.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+  <v-container fluid class="fill-height px-0 mb-4">
+
+    <div style="width: 100%" :overlay=false>
+      <div class="sticky full-width px-8">
+        <v-row>
+          <AlertMessage v-model="alert" :alertMessage="alertMessage" :alertType="alertType"></AlertMessage>
+        </v-row>
+        <v-row align="end" no-gutters justify="end" class="pb-6"
+               style="background-color:white;">
+
+          <PrimaryButton id="enter-data-search-action" :secondary="true"
+                         class="mx-3" :disabled="false"
+                         text="Enter Data & search" @click.native="openSearchDemographicsModal"></PrimaryButton>
+          <PrimaryButton id="issue-pen-action" class="mr-2" :disabled="isIssuePenDisabled" text="Issue new PEN"
+                         @click.native="issueNewPen()"></PrimaryButton>
+        </v-row>
+        <v-divider class="subheader-divider"/>
+        <v-row>
+          <v-col cols="12">
+            <v-row dense no-gutters class="mb-2">
+              <v-col cols="1"></v-col>
+              <v-col cols="1">Mincode</v-col>
+              <v-col cols="2">Legal Surname</v-col>
+              <v-col cols="2">Legal Given</v-col>
+              <v-col cols="2">Legal Middle</v-col>
+              <v-col cols="1">DC</v-col>
+              <v-col cols="1">Gen</v-col>
+              <v-col cols="1">Birth Date</v-col>
+              <v-col cols="1"></v-col>
+            </v-row>
+            <v-row dense class="mb-2">
+              <v-col cols="1"></v-col>
+              <v-col cols="1"><span :class="[demogValueMatched(STUDENT_DETAILS_FIELDS.MINCODE)?'font-weight-bold':'']"
+                                    style="word-wrap: break-word;font-size: medium">{{
```

Comment:
It would be better to use class to define css styles.

Reply:
ok I will update the code